lib/routes/charge/items/catalog_item.dart

```diff
@@ -83,12 +89,38 @@ class CatalogItem extends StatelessWidget {
     ]);
   }
 
-  String _formattedPrice() {
-    var formatter = CurrencyWrapper.fromShortName(_itemInfo.currency, accountModel);
-    return formatter.format(_itemInfo.price, includeCurencySuffix: true, removeTrailingZeros: true);
+  InlineSpan _buildCurrencySymbol() {
+    return _itemInfo.currency == "SAT"
+        ? WidgetSpan(
+            child: Container(
+              height: 20,
+              child: Align(
+                child: Text(_getSymbol(), // Icon
+                    style: TextStyle(fontSize: 13.4, fontFamily: 'SAT')),
+                alignment: Alignment.center,
```

Comment:
Custom fontFamily is used to display SAT icon.
Character for SAT icon has it's issues, such as it's alignment and font size is not consistent even with the font family(Ubuntu) it's derived from. 
The symbol is wrapped around with a center aligned container of height 20 to match style of price tags for other currencies.

Reply:
It is better to fix the font. @yaslama  is working on this. In the meanwhile can you use "S" character for satoshi and remove the work around?

Reply:
[Done](https://github.com/breez/breezmobile/pull/292/commits/9120cc013e7616ee53f12e4c7b3abba168d95cc6)